spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -20,26 +20,57 @@ def foobar_action
 end
 
 RSpec.describe Hq::PartnersController, type: :controller do
+
   before :each do
     sign_in instance_double(AdminUser)
-    @partner = build_stubbed :partner
-    expect(Partner).to receive(:find).and_return(@partner)
   end
 
-  specify 'editing renders the edit view' do
-    get :edit, id: @partner.id
-    expect(response).to render_template 'edit'
-  end
+  context '#new and #create' do
+    
+    before :each do
+      @partner = FactoryGirl.build_stubbed(:partner)
+    end
+    
+    specify 'successful create redirects to the show view' do
+      expect_any_instance_of(Partner).to receive(:save) do |_self|
+          _self.id = @partner.id
+          true
+      end
+      post :create, partner: @partner.attributes
+      expect(response).to redirect_to hq_partner_path(@partner.id)
+    end
```

Comment:
I'd seen that too, and think it's fine: the controller does exactly what it's told - redirect to whatever `@partner` is set to. We'll never see this output anyway.